packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -163,7 +163,7 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         override
         public
         returns (
-            bytes memory
+            bytes memory _returndata
```

Comment:
Is anybody opinionated on what this should be named? It needs a name for the geth ABI package to work

Reply:
The value you added here isn't currently being used. You'd ether want to update the variable below from `returndata` to `_returndata` or just remove the variable declaration in generally. Personally I'm very down to just remove the variable declaration